client/client.go

```diff
@@ -62,17 +59,9 @@ func (c *RemoteServer) getURL(path string) string {
 func (c *RemoteServer) doAction(action string, req []byte) ([]byte, error) {
 	buf := bytes.NewBuffer(req)
 	url := c.getURL("/" + action)
-	b := backoff.Backoff{}
```

Comment:
Instead of just removing the retry logic entirely, can we limit it to three or five attempts?